test/v1/connectors/PensionChargesConnectorSpec.scala

```diff
@@ -16,6 +16,7 @@
 
 package v1.connectors
 
+import data.AmendPensionChargesData.pensionCharges
```

Comment:
most of the val definitions that are in this test spec are also in the appropriate data object (either Retrieve or Amend). Could you tidy this file up and just use the ones in the data packages?